story_schema.py

```
@@ -0,0 +1,554 @@
+    if not why:
+        # Try to extract from CodeRabbit summary
+        if body and "Summary by CodeRabbit" in body:
+            lines = body.split("\n")
+            in_summary = False
+            in_new_features = False
+            for line in lines:
+                line = line.strip()
+                if "Summary by CodeRabbit" in line:
+                    in_summary = True
+                    continue
+                if in_summary and (line.startswith("- New Features") or line.startswith("‑ New Features")):
```

Ruff (0.12.2)

String contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)